impl.go

```diff
@@ -128,11 +136,13 @@ func typeSpec(path string, id string, srcDir string) (Pkg, *ast.TypeSpec, error)
 				if spec.Name.Name != id {
 					continue
 				}
-				return Pkg{Package: pkg, FileSet: fset}, spec, nil
+				return Pkg{Package: pkg, FileSet: fset},
```

Comment:
This return has gotten complicated. To it easier to read, let's break it apart onto multiple lines. (Something like `p := Pkg{...}; s := Spec{...}; return p, s, nil`)

Reply:
Done.

Reply:
Done.